modules/Layout/Boilerplate/Boilerplate.module.scss

```diff
@@ -40,13 +40,18 @@
 
 .about {
     margin-bottom: $spacing-5;
+
+    * {
```

Comment:
There's a bunch of options (headings 1 to 6, quote etc.) so this felt like the simplest solution that does not affect anything else.

Reply:
Should be fine, but I would go with `color: inherit`, just so we don't need to maintain it in multiple places.